package.json

```
@@ -81,7 +81,7 @@
     "eslint-plugin-jsdoc": "^39.6.2",
     "eslint-plugin-n": "^15.5.1",
     "eslint-plugin-promise": "^6.1.1",
-    "html-validate": "7.5.0",
+    "html-validate": "7.10.0",
```

Probably want to make this update-able locally?

Not blocking though

```suggestion
    "html-validate": "^7.10.0",
```